pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -702,6 +702,13 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 				return err
 			}
 		}
+		if iface.Driver != "" {
+			if iface.Driver != "qemu" && iface.Driver != "vhost" {
+				log.Log.Infof("The network interface driver type of %s is not supported. Silently ignored and fallback to default one.", iface.Name)
```

Comment:
We should return an error. If the unsupported interface type gets that far we would from my opinion just mask a software bug.

Reply:
Totally true, will fix it.